cdk_framework/EKS/lib/utils/validate-config-schema.ts

```diff
@@ -0,0 +1,56 @@
+//install using npm install ajv
+import Ajv, {DefinedError} from "ajv"
+
+const ajv = new Ajv({allErrors: true});
+require("ajv-errors")(ajv /*, {singleError: true} */)
+
+const schema = {
+    type: "object",
+    properties: {
+        clusters : {
+            type: "array",
+            items: {
+                type: "object",
+                properties: {
+                    name: {
+                        type: "string",
+                        errorMessage: {
+                            type: 'Name must be a string'
+                        }
+                    },
+                    version:  {
+                        type: "string",
```

Comment:
Should version be a string or number type?

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-eks-cluster.html#cfn-eks-cluster-version This is a link that Raphael sent me that seems to support this.